$OpenBSD: patch-src_tracker-extract_tracker-extract-mp3_c,v 1.4 2009/10/12 17:42:55 ajacoutot Exp $

- use glib's strndup(), instead of libc's version. Which may not be present
  on every system.

--- src/tracker-extract/tracker-extract-mp3.c.orig	Wed May 20 18:28:31 2009
+++ src/tracker-extract/tracker-extract-mp3.c	Mon Oct 12 19:24:57 2009
@@ -853,7 +853,7 @@ get_id3v24_tags (const gchar *data,
 							  NULL, NULL, NULL);
 					break;
 				case 0x03 :
-					word = strndup (&data[pos+11], csize-1);
+					word = g_strndup (&data[pos+11], csize-1);
 					break;
 
 				default:
